### ISSUE TITLE
reading speed ?

### ISSUE THREAD

**Issue by elz64** [aFarkas/jMediaelement#11]
I'm looking for something that could slow or accelerate the reading of a video.

Is there such a control ?

thank you

**Comment by aFarkas** [comment]
Do you mean the playbackRate of a video?

**Comment by elz64** [comment]
Yes I mean to have a slower or faster control for html5 video players 
(or not html5) like in dvd plyer where you can ply half speed or 10x 
faster for example.
